src/main/java/com/quizapp/enterprise/controllers/QuizController.java

```diff
@@ -28,7 +28,7 @@
 
 
     @RequestMapping("/quizzes")
-    public String userQuizes(Model model, Principal principal){
+    public String userQuizes(Model model, Principal principal) throws Exception {
```

Comment:
related to respective handling of null pointer exception change

Reply:
See comments above.